packages/server/src/api/index.ts

```diff
@@ -62,6 +63,8 @@ if (apiEnabled()) {
     .use(auth.auditLog)
     // @ts-ignore
     .use(migrations)
+    // @ts-ignore
+    .use(cleanup)
```

Comment:
When does this middleware get executed? Right now based on this flow, it looks like it'll happen right before any controller is actually called, so will it not just end up building another isolate-vm?